fusion_toolbox/reaction_rate/bosch_hale.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+class Bosch_Hale_reactivity:
+    """Class for Bosch Hale reactivity (H.-S. Bosch and G.M. Hale 1992 Nucl. Fusion 32 611)
+
+    Parameters
+    ----------
+    reaction : string
+        String name for the fusion reaction
+
+    """
+    known_reactions = ["DDn", "DDp", "DTn", "D3Hep"]
+
+    _DDn_Cs   = [5.43360e-12, 5.85778e-3, 7.68222e-3, 0.0, -2.96400e-6, 0.0, 0.0]
+    _DDp_Cs   = [5.65718e-12, 3.41267e-3, 1.99167e-3, 0.0, 1.05060e-5, 0.0, 0.0]
+    _DTn_Cs   = [1.1302e-9, 1.51361e-2, 7.51886e-2, 4.60643e-3, 1.35e-2, -1.0675e-4, 1.366e-5]
+    _D3Hep_Cs = [5.51036e-10, 6.41918e-3, -2.02896e-3, -1.91080e-5, 1.35776e-4, 0.0, 0.0]
```

Comment:
This is still not PEP8 compliant.

You can use a linter to identify the PEP8 issues:

```
python -m pylint [path_to_your_script]
```

Reply:
Fixed with https://github.com/tmarkj/fusion-toolbox/commit/1b1e5e104cc9fde8318d420baaee8e372f3d3460